splits/split_misc.sh

```diff
@@ -43,12 +43,16 @@ echo "Flink distribution directory: $FLINK_DIR"
 
 # run_test "<description>" "$END_TO_END_DIR/test-scripts/<script_name>"
 
+run_test "Flink CLI end-to-end test" "$END_TO_END_DIR/test-scripts/test_cli.sh"
```

Comment:
please also update `split_misc_hadoopfree` to include all tests independent of hadoop. I've copied `split_misc.sh` for now since it's the easiest&fastest option.

Reply:
I will copy some tests but I think we need to further split `split_misc_hadoopfree` because we run out of time otherwise.